src/content/blog/2024-09-30-dwmrc-and-ugrc-release-map-to-help-utahns-recycle-better.mdx

```diff
@@ -0,0 +1,30 @@
+
+Let’s start with the process. First, UGRC extracts the data from DWMRC’s SalesForce database, which houses the reporting forms sent in by municipal and state waste management facilities. Since every waste management facility in Utah is required to report their data on standardized forms, there was a large volume of data to work with, and more is added every week. 
+
+Once the tabular data are organized and cleaned, we use pandas scripts to perform calculations and gather statistics from the reports. These scripts run fully automated so that the statistics will be updated as new data are entered into the database. The fully automated process saves time, money, and helps to prevent human error.
+
+Even if you don’t have a haunted vase from a long-lost relative lying around (perhaps we should all check?), there are plenty of reasons why having access to this data is a good thing. First and foremost, the map serves as a reference for where you can dispose of any other haunted articles you have in your possession and regular items such as paint cans, batteries, and furniture. The map also points out recycling centers and has been used by organizations such as the Utah Recycling Alliance to help spread awareness about recycling efforts ([2](https://www.utahrecyclingalliance.org/post/utah-s-new-recycling-data-initiative)). Just knowing where and how to recycle can help people to actually do it. 
+
+Another benefit of this project is transparency. Utahns deserve to know how the government is using taxpayer’s dollars to promote effective waste management practices in our state. With these data products, the average person can see important metrics such as the percentage of waste materials that were recycled in Utah and how many gallons of used oil were collected for recycling. Having these metrics publicly available adds an extra layer of accountability for waste management facilities, DWMRC.
+
+This project is one of many collaborative efforts between UGRC and other state and federal agencies. At UGRC, our mission is to “encourage and facilitate the effective use of geospatial information and technology for Utah.” ([3](https://gis.utah.gov/about/) Working with DWMRC to produce these data products is just one of the many ways we advocate for data-driven decision making and public access to geospatial data in Utah. 
+
+You can learn more about our other projects on UGRC's [blog](https://gis.utah.gov/blog/) and in our [quarterly newsletter](https://cdn.forms-content.sg-form.com/61e82d28-f0d5-11ec-847c-a26809937a38). For more information on waste management in Utah and these data products in particular, please reach out to Stevie Norcross at DWMRC (stevienorcross@utah.gov) or Jake Adams at UGRC (jdadams@utah.gov).
```

Comment:
The comment that had this was resolved. adding a new one :)

1. emails should be linked from contacts and not placed directly in content [example](https://github.com/agrc/gis.utah.gov/blob/main/README.md#contact-information)